src/router/metadata/cap.template.ts

```diff
@@ -3,21 +3,26 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-import { FhirVersion } from 'fhir-works-on-aws-interface';
+import { FhirVersion, ProductInfo } from 'fhir-works-on-aws-interface';
 
-export default function makeStatement(rest: any, orgName: string, url: string, fhirVersion: FhirVersion) {
+export default function makeStatement(rest: any, productInfo: ProductInfo, url: string, fhirVersion: FhirVersion) {
     const cap: any = {
         resourceType: 'CapabilityStatement',
+        name: productInfo.productSlug ?? 'FhirServerCapabilityStatement',
+        title: (productInfo.productName ?? 'Fhir Server') + ' Capability Statement',
+        description: productInfo.productDescription ?? `A FHIR ${fhirVersion} Server Capability Statement`,
+        purpose: productInfo.productPurpose ?? `A statement of this system's capabilities`,
+        copyright: productInfo.copyright ?? `Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.`,

```

Comment:
hmmm definitely do not want Amazon as the default copyright -- is it fine to have this as null?

Reply:
Yes, absolutely.

Reply:
```suggestion
        copyright: productInfo.copyright ?? '',

```